sriovnet.go

```diff
@@ -413,3 +414,24 @@ func GetVfIndexByPciAddress(vfPciAddress string) (int, error) {
 	}
 	return -1, fmt.Errorf("vf index for %s not found", vfPciAddress)
 }
+
```

Comment:
@moshe010 
1. please add example comment section to describe input.
2. drop vf prefix from the input variable, as this is generic for any pci device.
3. Since API has to return multiple netdevices, you should say GetNetdevicesForPciDev().

Keep in mind that netdevice names can change with the udev rules in scheme dynamically.
So be prepare to return indices in future.